modules/gltf/src/lib/extensions/EXT_structural_metadata.ts

```diff
@@ -48,42 +48,49 @@ export function getPropertyTableFromExtStructuralMetadata(
   extension: GLTF_EXT_structural_metadata_GLTF,
   metadataClass?: string
 ): FeatureTableJson | null {
+  /**
+   * Note, 3dTiles is able to have multiple featureId attributes and multiple feature tables.
+   * In I3S we should decide which featureIds attribute will be passed to geometry data.
```

Comment:
What the explanation about I3S here? glTF is a lower level module. It doesn't know about I3S and 3DTiles. I would move this function to `tile-converter`.

Reply:
Done